### PR TITLE
Feature/age groups from tenant config

### PR DESCRIPTION
**Pull request by fxstr**

### CODE REVIEW

File: src/models/populationFilter/setupAgeGroups.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            // Only continue when all necessary data is available
+            if (statusIdentifier !== storeStatus.ready) return;
+
+            // Get tenantConfig.frontend.rda.ageGroups
+            const frontendConfig = tenantConfig.getConfig('frontend');
+            if (
+                !frontendConfig ||
+                !frontendConfig.rda ||
+                !frontendConfig.rda.ageGroups
+                // We cannot easily check if it's an array, as ageGroups is an observableArray
```
Comment (tobiaskneubuehler) [inline src/models/populationFilter/setupAgeGroups.js]: Array.isArray(frontendConfig.rda.ageGroups.slice()) this check should work

Comment (fxstr) [reply src/models/populationFilter/setupAgeGroups.js]: Yes and no. By using `.slice()` you already assume that ageGroups  an array. Is it not, your check will throw. The result would therefore be the same as without a check. :-D

File: src/infectApp.js [hunk 2]
```diff
@@ -32,6 +33,7 @@ import TenantConfigStore from './models/tenantConfig/TenantConfigStore.js';
 import notificationSeverityLevels from './models/notifications/notificationSeverityLevels.js';
 import GuidelineSelectedFiltersBridge from
     './models/guidelineSelectedFiltersBridge/GuidelineSelectedFiltersBridge.js';
+import Store from './helpers/store.js';
```
[the remark this answers is not in the document]
Comment (fxstr) [reply src/infectApp.js]: True that, thanks!

File: src/models/populationFilter/setupAgeGroups.js (hunk 1 above)
Comment (tobiaskneubuehler) [reply src/models/populationFilter/setupAgeGroups.js]: good point ;-)

**Pull request merged**